### PR TITLE
docs - add landing page for file_fdw module

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- landing page for file_fdw contrib module.  includes greenplum considerations (mpp_execute) and xrefs to postgres file_fdw page.

question:  do we plan to include this contrib module (or postgres_fdw) in 6.0?  if so, will backport this PR to the 6.0 docs.

doc review site link:  http://docs-lisa-review.cfapps.io/6-0/ref_guide/modules/file_fdw.html

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/modules/file_fdw.xml [hunk 1]
```diff
@@ -39,17 +39,27 @@
...
-        setting <codeph>mpp_execute 'any'</codeph>.
-    <note>While Greenplum Database will allow you to create a foreign server or
-      foreign table that specifies <codeph>mpp_execute 'any'</codeph>, it will
-      return an error when you <codeph>SELECT</codeph> the foreign table.</note></p>
+OPTIONS (format 'csv', filename '/data/csvtextfile&lt;SEGID>.csv', mpp_execute 'all segments');</codeblock></p>
+      <p>And the file on the server system must include the segment identifier as
+        well. For example:
+        <codeblock>/data/csvtextfile0.csv</codeblock></p>
+      <note>The Greenplum Database <codeph>file_fdw</codeph> module does not support
+        the <codeph>mpp_execute 'any'</codeph> setting. While Greenplum Database will
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/file_fdw.xml]: While -> Although
will allow -> allows

File: gpdb-doc/dita/ref_guide/modules/file_fdw.xml [hunk 2]
```diff
@@ -39,17 +39,27 @@
...
+        allow you to create a foreign server or foreign table that specifies
+        <codeph>mpp_execute 'any'</codeph>, it will return an error when you
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/file_fdw.xml]: will return -> returns

File: gpdb-doc/dita/ref_guide/modules/file_fdw.xml [hunk 3]
```diff
@@ -39,17 +39,27 @@
...
+        <codeph>SELECT</codeph> the foreign table.</note>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/file_fdw.xml]: SELECT -> query

**Comment by lisakowen** [comment]
thank you for reviewing, @adam8157.  do you happen to know if this module will be included in 6.0?

**Comment by dyozie** [comment]
@ivannovick @Eulerizeit - can either of you confirm whether the file_fdw module is going to be included with 6.x GA?

**Comment by Eulerizeit** [comment]
>@ivannovick @Eulerizeit - can either of you confirm whether the file_fdw module is going to be included with 6.x GA?

I don't think we should be shipping it. I'll check with @kmacoskey We are testing it in order to make sure we don't break the API, I had no intention of us shipping it.

**Comment by lisakowen** [comment]
closing since we are not shipping this.

**Pull request closed without merging**